### PR TITLE
Modified appliance_networking test

### PR DESCRIPTION
**Pull request by prabinovRedhat**
Modified appliance_networking test to run on real vCenter env

### CODE REVIEW

**Comment by prabinovRedhat** [comment]
cc: @shellymiron @anna-savina

**Review by anna-savina — APPROVED** [review]
LGTM except small suggestion

File: tests/integration/targets/vmware_rest_appliance/tasks/appliance_networking.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        var: set_network_config
+
+    - name: Assert network configuration was updated
+      ansible.builtin.assert:
+        that:
+          - set_network_config is defined
+          - set_network_config.changed
+        fail_msg: "Failed to update the network configuration to disable IPv6."
+
+    - name: Refresh the current network configuration
```
[the remark this answers is not in the document]
Comment (prabinovRedhat) [reply tests/integration/targets/vmware_rest_appliance/tasks/appliance_networking.yml]: thanks, changed

**Review by shellymiron — APPROVED** [review]

**Pull request merged**